python/lsst/eotest/image_utils.py

```diff
@@ -240,6 +348,58 @@ def bias_image(im, overscan, dxmin=5, dxmax=2, statistic=np.mean, bias_method='r
         imarr[row] += values[row]
     return biasim
 
+
+def bias_image_col(im, overscan, dxmin=5, dxmax=2, statistic=np.mean, bias_method='col', **kwargs):
+    """Generate a bias image containing the offset values calculated from
+    bias(), bias_col(), bias_func() or bias_spline().
+
+    Args:
+        im: A masked (lsst.afw.image.imageLib.MaskedImageF) or unmasked
+            (lsst.afw.image.imageLib.ImageF) afw image.
+        overscan: A bounding box for the parallel overscan region.
+        dxmin: The number of rows to skip at the beginning of the parallel
+            overscan region.
+        dxmax: The number of rows to skip at the end of the parallel overscan region.
+        statistic: The statistic to use to calculate the offset for each row.
+        bias_method: Either 'mean', 'col', 'func' or 'spline'.
+
+    Keyword Arguments:
+        fit_order: The order of the polynomial. This only needs to be specified when
+            using the 'func' method. The default is: 1.
+        k: The degree of the spline fit. This only needs to be specified when using
+            the 'spline' method. The default is: 3.
+        s: The amount of smoothing to be applied to the fit. This only needs to be
+            specified when using the 'spline' method. The default is: 18000.
+        t: The number of knots. If None, finds the number of knots to use for a given
+            smoothing factor, s. This only needs to be specified when using the 'spline'
+            method. The default is: None.
+
+    Returns:
+        An image with size equal to the input image containing the offset level.
+    """
+    if bias_method not in ['mean', 'col', 'func', 'spline']:
```

Comment:
Why do you change the code to do the `bias_method.lower()` conversion above, but not here?   I don't think it's necessary to do that conversion.   The client code should just give the right string.
Also, it seems you need to add `'none'` (or better `None`) as an option here, although since you have the test below whether to subtract `bias_image_col`, maybe you don't need this option.  In that case, I'd delete the `dummy_none` function below.

Reply:
Ok, I'll remove the dummy_none  See above for why I had the 'lower()' above but now here.